services/movie_session.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime
+
+import init_django_orm  # noqa: F401
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(
+        session_date: str = None
+) -> object:
+    if session_date:
+        try:
+            return MovieSession.objects.filter(show_time__date=session_date)
+        except MovieSession.DoesNotExist:
+            return MovieSession.objects.none()
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(
+        movie_session_id: int
+) -> object:
+    try:
+        return MovieSession.objects.get(id=movie_session_id)
+    except MovieSession.DoesNotExist:
```

Comment:
Explicit error handling is not expected in this task, remove it here and in the other places as well